internal/install/execution/install_status.go

```diff
@@ -43,6 +43,7 @@ type InstallStatus struct {
 	InstalledPacks            []*ObservabilityPackStatus `json:"packslInstalled"`
 	RedirectURL               string                     `json:"redirectUrl"`
 	HTTPSProxy                string                     `json:"httpsProxy"`
+	UpdateRequired            bool
```

Comment:
❗ Reminder: get clarity on the [`SetVersions()`](https://github.com/newrelic/newrelic-cli/pull/1036/files#diff-a4104ffcec79e09559961647e745b2de0686e79944e35d2b8a7de7811830c714R387-R394) method

Reply:
After digging a bit, we can definitely use `cli.Version()` here as well. I've updated accordingly.